dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/deployments/DeploymentTest.java

```diff
@@ -266,13 +296,30 @@ private void deployApp(Archive<?> archive, Consumer<ServerConfiguration> configM
         server.updateServerConfiguration(config);
         server.waitForConfigUpdateInLogUsingMark(Collections.singleton(getName(archive)));
     }
+    
+    private void deployLooseApp(String fileName, String type) throws Exception {
```

Comment:
Potentially, but it needs quite a lot of extra stuff (files already deployed to the correct locations etc.) I'd consider putting in a common util class if we end up needing to do the same thing again.

Fortunately, testing with loose config doesn't come up very much, but if we had to do it more for some reason, I'd probably want to also make a utility which would handle copying the files around and writing the loose config xml file so that there wasn't so much manual copying and supporting files for each test.